features/legacy/frontend/cart_promotions_percentage.feature

```diff
@@ -8,12 +8,12 @@ Feature: Checkout percentage discount promotions
         Given store has default configuration
         And the following promotions exist:
             | code | name    | description                                   |
-            | PR1  | 3 items | 25% Discount for orders with at least 3 items |
+            | PR1  | 5 items | 25% Discount for orders with at least 5 items |
             | PR2  | 300 EUR | 10% Discount for orders over 300 EUR          |
-        And promotion "3 items" has following rules defined:
-            | type       | configuration        |
-            | item_count | Count: 3,Equal: true |
-        And promotion "3 items" has following actions defined:
+        And promotion "5 items" has following rules defined:
+            | type          | configuration        |
+            | cart quantity | Count: 5,Equal: true |
```

Comment:
Just out of sheer curiosity, why this one `cart` is in lower case while all others are in upper?